components/community/Header.tsx

```diff
@@ -18,7 +18,7 @@ interface HeaderProps {
 export default function Header({
   // eslint-disable-next-line no-unused-vars, unused-imports/no-unused-vars
   className = ''
-}: HeaderProps): JSX.Element {
+}: HeaderProps): React.ReactNode {
```

Comment:
@anshgoyalevil We should not use `React.ReactNode` here because it's being used inside the JSX in `Hero.tsx`. Due to this, using `React.ReactNode` is causing a build error.

Reply:
I have changed this back to `JSX.Element` for now, let me know if there is any other option.

Reply:
Maybe you should then remove it all together, as suggested by Akshat